examples/hooks/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useBreadcrumb, UseBreadcrumbProps } from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type BreadcrumbProps = React.ComponentProps<'div'> & UseBreadcrumbProps;
+
+export function Breadcrumb(props: BreadcrumbProps) {
+  const { items, refine, createURL } = useBreadcrumb(props);
+
+  return (
+    <div
+      className={cx(
+        'ais-Breadcrumb',
+        items.length === 0 && 'ais-Breadcrumb--noRefinement',
+        props.className
+      )}
+    >
+      <ul className="ais-Breadcrumb-list">
+        <li
+          className={cx(
+            'ais-Breadcrumb-item',
+            items.length === 0 && 'ais-Breadcrumb-item--selected'
+          )}
+        >
+          <a
+            href={createURL(null)}
```

Comment:
We've been using `undefined` to reflect that the refined value must be unset.

```suggestion
            href={createURL(undefined)}
```

Reply:
why did you resolve @FabienMotte? it seems like you still use `null`?

Reply:
@Haroenv Yes indeed, I discussed about it with @francoischalifour and we decided to keep it as it is, to use `undefined`, we should first update the typing here: https://github.com/algolia/instantsearch.js/blob/master/src/connectors/breadcrumb/connectBreadcrumb.ts#L30
WDYT?

Reply:
I think we can use `undefined` with a `ts-expect-error` and fix the typing in IS.js.

Reply:
hmm, or we use `null` in the InstantSearch.js component too? I guess it doesn't matter too much for now as it just checks if the value is falsy https://github.com/algolia/instantsearch.js/blob/master/src/connectors/breadcrumb/connectBreadcrumb.ts#L131

Reply:
I'll go with @francoischalifour proposal, thanks!